server/svix-server/src/main.rs

```diff
@@ -98,13 +102,18 @@ async fn main() {
         }
     };
 
+    let redis_cache = redis_pool
+        .as_ref()
+        .map(|pool| RedisCache::new(pool.clone()));
```

Comment:
It's fine for this PR, but we probably need to improve it in a subsequent PR:
We should make the cache object always exist, and just have different backends like Queue.

The backends can be:
1. Redis
2. Memory
3. No-op (though tbh, probably memory with size 0 is a no-op).

This also makes the code using it much cleaner because it means the cache object always exists, it just happens to always cache miss with it's a no-op, or etc.